exercises/concept/boutique-inventory/.docs/introduction.md

```diff
@@ -0,0 +1,54 @@
+Enumeration is the act of stepping through a collection (`Array`, `Hash`, etc) and performing some action on each object.
+
+Enumeration is a key concept in Ruby and is used for sorting (`sort_by`), grouping (`group_by`), mapping (`map`), reducing (`reduce`), and much more. 
+You'll most frequently see enumeration as the idiomatic way iterating through collections rather than using loops.
```

Comment:
```suggestion
You will most frequently see enumeration as the idiomatic way iterating through collections rather than using loops.
```

Avoiding contractions, due to international/multilingual audience.